### DISCUSSION TITLE
Stuck on v2.0.0b11

### DISCUSSION THREAD

**Discussion by freeman59** [Tasshack/dreame-vacuum#494] [General]
Hello

i'm contacting you because since v2.0.0b11 I cannot update the integration. I click on HACS section, update DREAM VACUM but it still  goes around in circles and nothing works ... Waited like 20 minutes and still same problem. I have b12, b13 and b14 showed as new version and tried several times since weeks ...

Do you know what is the problem ? I have so much integration but only this one doesn't work. 

Thank you !

**Comment by Tasshack** [comment]
https://github.com/Tasshack/dreame-vacuum/issues/429
